client/gtk2/ibusimcontext.c

```diff
@@ -1529,6 +1530,69 @@ _set_cursor_location_internal (IBusIMContext *ibusimcontext)
 
     area = ibusimcontext->cursor_area;
 
+#if GTK_CHECK_VERSION (3, 98, 4)
+    graphene_point_t p;
+
+    if (!gtk_widget_compute_point (ibusimcontext->client_window,
+                                   GTK_WIDGET (gtk_widget_get_root (ibusimcontext->client_window)),
+                                   &GRAPHENE_POINT_INIT (area.x, area.y),
+                                   &p))
```

Comment:
Since the `if` condition is described with multi lines, please enclose graphene_point_init with `{}`.
```
+                                   &p)) {
         graphene_point_init (&p, area.x, area.y);
+    }

```